.changeset/heavy-plants-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-user-settings': patch
+---
+
+Added experimental support for declarative integration
```

Comment:
```suggestion
Added experimental support for declarative integration via the `/alpha` export.
```

I like calling this out just to make it clear where the change is being made so users wouldn't go looking for it in the wrong place